cognite/client/cognite_client.py

```diff
@@ -52,7 +52,7 @@ class CogniteClient:
 
                 from cognite import CogniteClient
                 client = CogniteClient()
-                res = client.time_series.get_timeseries()
```

Comment:
This is a breaking change...? Should we revert this change?

Reply:
it's in a docstring. fixes outdated documentation.

Reply:
see #226